docs/integrations/databases/supabase.mdx

```diff
@@ -3,170 +3,666 @@ title: Integrate Supabase with Clerk
 description: Learn how to integrate Clerk into your Supabase application.
 ---
 
-# Integrate Supabase with Clerk
 
-The first step is to create a new Clerk application from your Clerk Dashboard if you haven’t done so already. You can choose whichever authentication strategy and social login providers you prefer. For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+    },
+  ]}
+>
 
-After your Clerk application has been created, go to the Clerk Dashboard and navigate to the **[JWT Templates](https://dashboard.clerk.com/last-active?path=jwt-templates)** page. Click on the **New template** button to create a new template based on Supabase.
+- Use Clerk to authenticate access to your Supabase data
+- Access Clerk user IDs in your Supabase RLS policies
+- Customize a JWT template to suit your use-case with Supabase
 
-<Images 
-  width={3024}
-  height={1652}
-  src="/docs/images/integrations/supabase/jwt-template.webp"
-  alt="The JWT Templates page in the Clerk Dashboard. The 'New template' button was clicked, and a pop up titled 'New JWT template' is shown. The 'Supabase' template is hovered over."
-/>
+</TutorialHero>
 
-Once the Supabase template is created, you will be redirected to the template's page. You can now configure the template to your needs.
+Integrating Supabase with Clerk enables you to implement Supabase's robust storage options while leveraging Clerk's authentication, prebuilt components, and webhooks.
 
-<Images 
-  width={3024}
-  height={1652}
-  src="/docs/images/integrations/supabase/create-template.webp"
-  alt="The 'Create new template' page of the JWT Templates page in the Clerk Dashboard."
-/>
+The following steps will teach you how to integrate Supabase with Clerk by creating [Row Level Security](https://supabase.com/docs/guides/auth/row-level-security) (RLS) policies that show users content associated with their account.
 
-The Supabase template will pre-populate the default claims required by Supabase. You can include additional claims as necessary. [Shortcodes](/docs/backend-requests/making/jwt-templates#shortcodes) are available to make adding dynamic user values easy.
+<Steps>
+ 
+### Add `user_id` to your Supabase tables
```

Comment:
Just want to make the callout that they shouldn't have to name it "user_id" -- they could name it whatever they'd like as long as they make sure when they implement the policy, they point to the correct property. 

Unsure if we need to make that callout, but wanted to make sure we're aware.